extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java

```diff
@@ -163,8 +165,20 @@ public static WrapperExtensionContext buildContext(
         var policyMappingService = new PolicyMappingService();
         var offeringService = new OfferingService(assetIndex, policyDefinitionStore,
                 contractDefinitionStore, policyMappingService);
+
+        transformerRegistry.register(new PermissionToPermissionDtoTransformer());
+        transformerRegistry.register(new PolicyToPolicyDtoTransformer());
+        transformerRegistry.register(new ContractAgreementToContractAgreementDtoTransformer());
+        transformerRegistry.register(new DataRequestToDataRequestDtoTransformer());
+        transformerRegistry.register(new ContractNegotiationToContractNegotiationOutputDtoTransformer());
+        transformerRegistry.register(new TransferProcessToTransferProcessOutputDtoTransformer());
+        var consumptionService = new ConsumptionService(contractNegotiationService, transferProcessService,
+                contractNegotiationStore, transferProcessStore, transformerRegistry, policyMappingService);
+
+        negotiationObservable.registerListener(new ContractNegotiationConsumptionListener(consumptionService));
```

Comment:
As long as we can we should try to avoid using the transformer registry to allow to see the code flow, calling functions depending on what's necessary, rather than registering global resources / mappings.

violates clean code / YAGNI